packages/client-python/src/rocketride/cli/commands/runtime.py

```
@@ -0,0 +1,1107 @@
+                print(f'No instance found with id: {instance_id}')
+                print('Use "rocketride runtime install" to create a new instance first.')
+                return 1
+
+            # Docker instances use Docker SDK for start
+            inst_type = existing.get('type', 'Local')
+            if inst_type == 'Docker':
+                print(f'Starting Docker runtime {instance_id}...')
+                docker = DockerRuntime()
+                try:
+                    docker.start(instance_id)
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)